### PR TITLE
NXP-21216: delete temporary files and folder after video conversion.

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

File: nuxeo-platform-video-convert/src/main/java/org/nuxeo/ecm/platform/video/convert/BaseVideoConversionConverter.java [hunk 1]
```diff
@@ -82,6 +86,7 @@
         }
         // delete the file as we need only the path for ffmpeg
         outFile.delete();
+        Framework.trackFile(outFile, this);
```
Comment (efge) [inline nuxeo-platform-video-convert/src/main/java/org/nuxeo/ecm/platform/video/convert/BaseVideoConversionConverter.java]: `this` is not the proper marker to use to track the file, its lifespan is too long (it lives in the conversion service as long as Nuxeo is running). You want as a marker something that's freed when the file is not needed anymore.

Comment (richardsd) [inline nuxeo-platform-video-convert/src/main/java/org/nuxeo/ecm/platform/video/convert/BaseVideoConversionConverter.java]: your comment makes sense.

**Comment by funshodavid** [comment]
:+1:

**Comment by efge** [comment]
👍

**Pull request closed without merging**